lib/fs.js

```diff
@@ -1742,6 +1742,13 @@ function WriteStream(path, options) {
   this.mode = options.hasOwnProperty('mode') ? options.mode : 438; /*=0666*/
 
   this.start = options.hasOwnProperty('start') ? options.start : undefined;
+
+  if (options.hasOwnProperty('autoClose')) {
```

Comment:
For consistency with the lines above it, this should probably use a ternary operator.